package.json

```diff
@@ -51,7 +51,7 @@
     "@kibertoad/biome-config": "^1.2.1",
     "@types/node": "^22.15.30",
     "@vitest/coverage-v8": "^3.2.2",
-    "fastify": "^5.3.3",
+    "fastify": "^5.4.1",
```

Comment:
peerDependency also needs to be bumped

Reply:
I was hesitant to bump the peerDependency for the following reason:
Currently, it specifies [`fastify ^5`](https://github.com/turkerdev/fastify-type-provider-zod/blob/c035e72ef76344d69660fcb6d951ae5c13ea1c03/package.json#L44), and updating it would require a major version bump. 
Since this change only affects TypeScript types and doesn't involve any runtime fixes I felt it wasn't worth triggering a major release for it.

No problem changing it though!